tasks/orca_catalog_reporting/orca_catalog_reporting.py

```diff
@@ -215,24 +216,25 @@ def handler(
         See schemas/output.json
         Or, if an error occurs, see create_http_error_dict
     """
-    LOGGER.setMetadata(event, context)
-
     try:
-        with open("schemas/input.json", "r") as raw_schema:
-            schema = json.loads(raw_schema.read())
+        LOGGER.setMetadata(event, context)
 
-        validate = fastjsonschema.compile(schema)
-        validate(event)
-    except JsonSchemaException as json_schema_exception:
-        return create_http_error_dict(
-            "BadRequest",
-            HTTPStatus.BAD_REQUEST,
-            context.aws_request_id,
-            json_schema_exception.__str__(),
-        )
+        try:
```

Comment:
I think we should be able to remove this try and move the except block to be next to the general except block.
See https://docs.python.org/3.7/tutorial/errors.html#handling-exceptions.